### DISCUSSION TITLE
It is possible to move a shape and a table?

### DISCUSSION THREAD

**Discussion by YRobles** [ShapeCrawler/ShapeCrawler#443] [Q&A]
Hi!

I'm trying to move a table and a shape. I've tried to use X and Y properties, but in a shape nothing happened, and in a table it throw and exception.
Is there any other way to move the place of a table and a shape in a slide?

Thanks in advanced.

**Comment by AdamShakhabov** [comment]
Hello,

thank you for the bug report.

Regarding the issue with **AutoShape moving**: I am unable to reproduce it. Could you please provide a presentation file with a shape that is not being moved? You can share it here or send it to theadamo86@gmail.com

Regarding the issue with **Table moving**: I am able to reproduce it. I will work on fixing it.

**Comment by AdamShakhabov** [comment]
Here is the fix for the Table moving [**0.41.2-beta.1**](https://github.com/ShapeCrawler/ShapeCrawler/files/10522143/ShapeCrawler.0.41.2-beta.1.nupkg.zip)
